### PR TITLE
Master size selection feedback for all cloud accounts

### PR DESCRIPTION
**Pull request by nikage**
- aws
- gce
- digital ocean

### CODE REVIEW

**Comment by egopher** [comment]
cc @josephkane

File: cmd/ui/assets/src/app/clusters/new-cluster/new-cluster.component.ts [hunk 1]
```diff
@@ -396,6 +421,49 @@ export class NewClusterComponent implements OnInit, OnDestroy {
...
+
+    const GCE_RECOMMENDATIONS = {
+      'n1-standard-1': 5,
+      'n1-standard-2': 10,
+      'n1-standard-4': 100,
+      'n1-standard-8': 250,
+      'n1-standard-16': 500,
+      'n1-standard-32': 1000,
+    };
+
```
Comment (josephkane) [inline cmd/ui/assets/src/app/clusters/new-cluster/new-cluster.component.ts]: I can't seem to get passed the machine config step e.g on gce, with 1 n1-standard-4 and 1 n1-megamem-96

image

File: cmd/ui/assets/src/app/clusters/new-cluster/new-cluster.component.ts [hunk 2]
```diff
@@ -1,23 +1,35 @@
...
+import { NodeProfileService } from '../node-profile.service';
+import { CLUSTER_OPTIONS } from './cluster-options.config';
 import {
   DEFAULT_MACHINE_SET,
   BLANK_MACHINE_TEMPLATE,
-}                                                                           from 'app/clusters/new-cluster/new-cluster.component.config';
-import { sortDigitalOceanMachineTypes }                                     from 'app/clusters/new-cluster/new-cluster.helpers';
-import { map }                                                              from 'rxjs/operators';
+} from 'app/clusters/new-cluster/new-cluster.component.config';
+import { sortDigitalOceanMachineTypes } from 'app/clusters/new-cluster/new-cluster.helpers';
```
Comment (josephkane) [inline cmd/ui/assets/src/app/clusters/new-cluster/new-cluster.component.ts]: these paths show errors, and for some reason make `currentmachine.recommendedNodesCount` on lines 410 and 413 error out. any chance these are invalid paths?

File: cmd/ui/assets/src/app/clusters/new-cluster/new-cluster.component.ts (hunk 1 above)
Comment (nikage) [reply cmd/ui/assets/src/app/clusters/new-cluster/new-cluster.component.ts]: fixed

File: cmd/ui/assets/src/app/clusters/new-cluster/new-cluster.component.ts (hunk 2 above)
Comment (nikage) [reply cmd/ui/assets/src/app/clusters/new-cluster/new-cluster.component.ts]: image

I've got no errors on this regard, are they still reproducible?

**Pull request merged**